extensions/ql-vscode/src/telemetry.ts

```diff
@@ -165,8 +165,9 @@ export class TelemetryListener extends ConfigListener {
       if (GLOBAL_ENABLE_TELEMETRY.getValue()) {
         // Extension won't start until this completes.
         result = await showBinaryChoiceDialog(
-          'Do we have your permission to collect usage data and metrics to help us improve CodeQL for VSCode? See [TELEMETRY.md](https://github.com/github/vscode-codeql/blob/main/TELEMETRY.md) for details of what we collect and how we use it.',
+          'CodeQL Extension: Do we have your permission to collect usage data and metrics to help us improve CodeQL for VSCode? See https://github.com/github/vscode-codeql/blob/main/TELEMETRY.md for details of what we collect and how we use it.',
```

Comment:
Either this needs to be `blob/main/extensions/ql-vscode/TELEMETRY.md` or we need to move the `md` file to the repo root. Probably the latter since we link to it elsewhere.

Also suggest either `Do we` -> `Does GitHub`, or `CodeQL extension by GitHub`.